samples/attested_tls/non_enc_client/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
```

Comment:
Looks like same issues here as the one above, I won't repeat further things.

Reply:
removed.